cfme/tests/configure/test_access_control.py

```diff
@@ -701,3 +701,35 @@ def test_superadmin_tenant_crud():
     with update(tenant):
         tenant.name = tenant.name + "edited"
     tenant.delete()
+
+
+@pytest.mark.uncollectif(lambda: version.current_version() < "5.5")
+def test_superadmin_tenant_project_crud():
+    """Test suppose to verify CRUD operations for CFME projects
+
+    Prerequisities:
+        * This test is not depending on any other test and can be executed against fresh appliance.
+
+    Steps:
+        * Create tenant
+        * Create project as child to tenant
+        * Update description of project
+        * Update name of project
+        * Delete project
+        * Delete tenant
+    """
+    tenant = ac.Tenant(
+        name='tenant1' + fauxfactory.gen_alphanumeric(),
+        description='tenant1 description')
+    tenant.create()
```

Comment:
Clearly said, and more correctly written, we need to ensure the objects created by the test get deleted. Usually we keep the code as I wrote in the previous comment (since deleting is not part of the test) but in this case when you test CRUD, this would be the perfect solution:

```python
# First define both tenant and project since just defining them has no effect to the UI
# Then this
@request.addfinalizer
def _delete_tenant_and_project():
    for item in {tenant, project}:
        if item.exists:
            item.delete()
```

The `.exists` check is necessary here. Then continue with creating both of those. :)